serverless/knative-serving/config-applications/multi-container-support-for-serving.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="multi-container-support-for-serving"]
+= Multi-container support for Serving
```

Comment:
Is "Serving" usually capitalized?

Reply:
Yeah, it's a component of Serverless. We have it this way often in the Serverless docs.